stratum/src/lib.rs

```diff
@@ -19,6 +19,8 @@
 extern crate json_tcp_server;
 extern crate jsonrpc_core;
 #[macro_use] extern crate log;
+extern crate env_logger;
+#[macro_use] extern crate lazy_static;
```

Comment:
`env_logger` and `lazy_static` are `cfg(test)` only, they shouldn't be linked in otherwise.